packages/updating-element/src/updating-element.ts

```diff
@@ -637,6 +691,20 @@ export abstract class UpdatingElement extends HTMLElement {
         (options.converter as ComplexAttributeConverter)?.toAttribute ??
         defaultConverter.toAttribute;
       const attrValue = toAttribute!(value, options.type);
+      if (
+        DEV_MODE &&
+        (this.constructor as typeof UpdatingElement).warnings!.has(
```

Comment:
I don't quite like the name `warnings` here. That sounds like some field I'd _read_ as a user to see what warnings were triggered. `showWarnings`?

Reply:
Changed to `enabledWarnings`.